setup.py

```diff
@@ -16,91 +16,86 @@
 ## Imports
```

Comment:
Would be nice to migrate this to pyproject.toml as well. Can use btrdb-python as a starting point most likely.